components/color-picker/index.jsx

```diff
@@ -57,6 +59,10 @@ const propTypes = {
 	 * Unique ID for component.
 	 */
 	id: PropTypes.string,
+	/**
+	 * Establishes directional context for component. Defaults to left-to-right.
+	 */
+	direction: PropTypes.oneOf([DIRECTIONS.LTR, DIRECTIONS.RTL]),
```

Comment:
color picker component does need the `direction` prop to decide the anchor point of the pop over, but does it need to be an official prop? I can remove this from here so that it doesn't show up in the docs. Is that preferred?

CC @minevskiy @andyouyang22 @interactivellama

Reply:
`Popover` alignment will be correctly flipped by its inner `Dialog` (see https://github.com/salesforce/design-system-react/pull/2077/). 
No need to adjust it here.

Reply:
@minevskiy did you mean to put this comment on line 400?

I should wait until #2077 and #2099 are merged and then rebase and merge this one. Need that also for that makeRtl reuse

Reply:
But again regarding this line. If I add it here it gets added to `components/component-docs.json`. I'm wondering if we want that.

Reply:
> did you mean to put this comment on line 400?
Since we don't need to flip Popover alignment, we also don't need `direction` prop.

Reply:
oh, lol, good point!! 👍